lib/setup.js

```diff
@@ -30,7 +30,7 @@ module.exports = function (options = {}) {
       plugins = [],
       env = [],
       cacheControl = {},
-      skipBootstrap = false
+      bootstrap = true
```

Comment:
will this be overwritten by a falsy value...?

Reply:
Yes, destructuring default values are only used if a value is not present in the object. Here is a little easy reading on destructuring: http://wesbos.com/destructuring-default-values/

The `Careful: null and undefined` section outlines this behavior, but since `false`/`null`/`undefined`/`0` are still values, they would be taken instead of `true`